### PR TITLE
Fix Issue 48466

### PR DESCRIPTION
**Pull request by labkey-alan**
#### Rationale
This PR fixes [Issue 48466](https://www.labkey.org/home/Developer/issues/issues-details.view?issueId=48466)

#### Related Pull Requests
* n/a

#### Changes
* select_rows: Set default max_rows to -1

### CODE REVIEW

File: labkey/query.py [hunk 1]
```diff
@@ -316,7 +316,7 @@ def select_rows(
     filter_array: List[QueryFilter] = None,
     container_path: str = None,
     columns=None,
-    max_rows: int = None,
+    max_rows: int = -1,
```
Comment (RosalineP) [inline labkey/query.py:319]: Down on line 376, should it be `if max_rows is not -1`?

Comment (RosalineP) [reply labkey/query.py:319]: Seperately, should the `@functools.wraps` part down on line 525 be updated to -1 as well?

**Review by RosalineP — APPROVED** [review]
Looks good other than the comments/questions I had! Should be simple changes that don't warrant another review, so I'm approving.

File: labkey/query.py (hunk 1 above)
Comment (labkey-alan) [reply labkey/query.py:319]: For 376 - No, it needs to remain that way or we'll never pass the default value.
For 525 - Yes, good catch.

**Pull request merged**